error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BigDecimalRules.java

```diff
@@ -50,17 +50,17 @@ BigDecimal after() {
     }
   }
 
-  /** Prefer {@link BigDecimal#valueOf(long)} over the associated constructor. */
-  // XXX: Ideally we'd also rewrite `BigDecimal.valueOf("<some-integer-value>")`, but it doesn't
+  /** Prefer {@link BigDecimal#valueOf(double)} over the associated constructor. */
+  // XXX: Ideally we'd also rewrite `new BigDecimal("<some-integer-value>")`, but it doesn't
```

Comment:
think this was meant here, not sure if this is a nice comment to keep around anyway, but I'll leave that up to you 😉

Reply:
Hmm, I wonder whether the comment meant a bit of both: if `"<some-integer-value>"` represents a value that can be expressed as a `long`, then passing in a `long`/`int` is nicer than a string. (But such a change can't be done using Refaster; it'd require a `BugChecker`.)

Reply:
Ah, I see that `BigDecimal.valueOf` doesn't exist. So certainly that was _not_ meant :smile: